src/js/models/v2/awardsV2/BaseIdv.js

```diff
@@ -66,6 +74,30 @@ BaseIdv.populate = function populate(data) {
         this.placeOfPerformance = placeOfPerformance;
     }
 
+    if (data.idv_dates) {
```

Comment:
This logic is duplicated after merging in dev (was added for the Dates and Agency/Recipient overview sections)